dttools/src/jx_querytools.c

```diff
@@ -0,0 +1,64 @@
+/*
+Copyright (C) 2020- The University of Notre Dame
+This software is distributed under the GNU General Public License.
+See the file COPYING for details.
+*/
+
+#include "jx.h"
+#include "jx_eval.h"
+#include "jx_getopt.h"
+#include "jx_parse.h"
+#include "jx_print.h"
+#include "stringtools.h"
+
+#include <errno.h>
+#include <stdio.h>
+#include <string.h>
+
+struct jx * jx_fetch_from_url ( const char *u ) {
```

Comment:
Do these placeholders devolve to the jx function implementations for select, project, fetch, etc?